app/routes/bbl.js

```diff
@@ -0,0 +1,9 @@
+import Ember from 'ember';
+import bblDemux from '../utils/bbl-demux';
+
+export default Ember.Route.extend({
+  model(params) {
```

Comment:
@allthesignals is this the right hook to use? I tried `beforeModel` but it appeared that the params weren't fully loaded.

Reply:
@pichot yes - beforeModel doesn't have the params object directly, you'd have to access it differently.